specifyweb/frontend/js_src/lib/components/Molecules/DateElement.tsx

```diff
@@ -33,9 +34,10 @@ export function DateElement({
         const now = new Date();
         const timeDifference = now.getTime() - dateObject.getTime();
 
-        if (timeDifference < 60000) intervalValue = 5000;
-        else if (timeDifference < 360000) intervalValue = 60000;
-        else intervalValue = 36000000;
+        if (timeDifference < 60 * MILLISECONDS)
+          intervalValue = 5 * MILLISECONDS;
+        else if (timeDifference < 6 * MINUTE) intervalValue = MINUTE;
+        else intervalValue = HOUR;
 
```

Comment:
now this is what I call readable 😊:

```suggestion
        const timeDifference = (now.getTime() - dateObject.getTime()) / MILLISECONDS;

        if (timeDifference < MINUTE)
          intervalValue = SECOND;
        else if (timeDifference < HOUR) intervalValue = MINUTE;
        else intervalValue = HOUR;
```